packages/realm/src/Collection.ts

```diff
@@ -37,14 +37,14 @@ export abstract class Collection<
 > implements Iterable<T>
 {
   /** @internal */
-  private listeners: Listeners<ChangeCallbackType, binding.NotificationToken>;
+  private listeners: Listeners<ChangeCallbackType, binding.NotificationToken, [string[] | undefined]>;
```

Comment:
Should we extract `[string[] | undefined]` into its own type?

Reply:
I'm fine with leaving it as-is in this particular case: I wouldn't say we need to export it, it's not duplicated all-over the code-base and don't expect it to change.